### DISCUSSION TITLE
기획 문서에 관한 질문들 (이벤트, 뉴스)

### DISCUSSION THREAD

**Discussion by mille-feu** [Tiny-Metro/Poly-Metro#59] [Q&A]
- 이벤트 UI의 미션에 관한 정보가 필요합니다
  - 해당 미션이 투자미션과 어떻게 다른지
  - 등장 조건이 어떻게되는지
- 뉴스에 관한 정보가 필요합니다
  - 각 이벤트의 등장 조건
  - 이벤트로 증감하는 수치의 합적용, 곱적용 여부

**Comment by cstrnull00** [comment]
* 이벤트 UI의 미션
  * 투자 미션 : 투자 탭에서 동시에 최대 3개까지 받을 수 있는 퀘스트의 일종이며, 달성 시 보상, 실패 시 패널티를 주는 능동적인 퀘스트.
  * 이벤트 : 무작위적으로 발생하는 효과로, 승객의 생성 속도 증감, 역의 파괴, 불만도의 증감 등이 변하는 수동적인 이벤트.
    * 이벤트는 동시에 하나만 발생
    * 이벤트는 2주 뒤부터 발생할 확률이 생기는데, 자세한 수치는 더 논의 후 전달하겠습니다.
* 이벤트의 효과는 [Document](https://github.com/Tiny-Metro/Tiny-Metro/wiki/Document:-%EC%9D%B4%EB%B2%A4%ED%8A%B8)에 서술되어있으며, 효과가 지속되는 기간 역시 명시되어 있습니다.
* 이벤트의 효과가 지속되는 동안은 새로운 이벤트가 생기지 않으므로, 이벤트로 증감하는 수치의 합적용, 곱적용에 대한 여부는 필요치 않습니다.
